### PR TITLE
add setters for tols in EvalTable and Validation

### PR DESCRIPTION
**Pull request by fschloesser**
I am unsure if the setters should go to the IPETEvalTable or the Experiment.

### CODE REVIEW

File: ipet/validation/Validation.py [hunk 1]
```diff
@@ -69,6 +69,29 @@ def __init__(self, solufilename : str = None, tol : float = DEFAULT_RELTOL, feas
...
+
+        Parameters
+        ----------
+
+        tol : float
+            new value for the tol for this validation
+        """
+        self.tol = tol
+
+    def set_feastol(self, feastol):
```
Comment (GregorCH) [inline ipet/validation/Validation.py]: Can you specify that both functions require a float attribute not only in the doc string?

File: ipet/evaluation/IPETEvalTable.py [hunk 2]
```diff
@@ -1094,6 +1116,10 @@ def validateData(self, df : DataFrame) -> DataFrame:
             v = Validation(self.validate)
         else:
             v = Validation(None)
+        if self.feastol:
+            v.set_feastol(self.feastol)
```
Comment (GregorCH) [inline ipet/evaluation/IPETEvalTable.py]: I would prefer to explicitly cast the `self.feastol` member to a float. in both calls. They might be strings for the evaluation object, if passed via xml.

Comment (fschloesser) [reply ipet/evaluation/IPETEvalTable.py]: I agree, maybe do that in the setters already instead of the calls?

Comment (GregorCH) [reply ipet/evaluation/IPETEvalTable.py]: No, not necessarily. This is simpler for exporting from and to xml to assume that all members of the evaluation are strings.

**Comment by fschloesser** [comment]
Okay, more comments?
And what happened to the travis?

**Pull request merged**